[OSDOCS-17003] IBM Z CQA work (#116100)
openshift/openshift-docs: modules/preferred-installation-requirements-ibm-z.adoc

[id="preferred-ibm-z-system-requirements_{context}"]
= Preferred {ibm-z-title} system environment

[role="_abstract"]

> The preferred system environment for running {product-title} version {product-version} on {ibm-z-name} hardware is as follows:

Author: ```suggestion
Use the following preferred system environment to run {product-title} version {product-version} on {ibm-z-name} hardware.
```
